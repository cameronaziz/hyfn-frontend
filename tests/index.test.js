import React from 'react';
import renderer from 'react-test-renderer';
import './adapter';

import Index from '../src/index';

describe('Index Component', () => {
  it('will match snapshot', () => {
    const component = renderer.create(<Index />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
