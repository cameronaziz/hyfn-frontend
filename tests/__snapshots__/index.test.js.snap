// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index Component will match snapshot 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src={
        Object {
          "process": [Function],
        }
      }
    />
  </header>
  <div>
    Loading
  </div>
</div>
`;
